CHANGELOG.md

```diff
@@ -31,6 +31,7 @@ changes.
 - JSON API: commands are once again read even if one hasn't responded yet (broken in 0.6.2).
 - Protocol: allow lnd to send `update_fee` before `funding_locked`.
 - pylightning: handle multiple simultanous RPC replies reliably.
+- pylightning: fixed an incompatibility when parsing responses from older clients (#2135).
```

Comment:
No, this was never broken in a release!  b2378654d70aea2a5b456f948eb67104dd50a5fe introduced it *after* 0.6.2.

Reply:
Ok, will amend to say it supports both :-)